aip/0235.md

```diff
@@ -0,0 +1,87 @@
+---
+aip:
+  id: 235
+  state: reviewing
+  created: 2019-06-18
+permalink: /235
+redirect_from:
+  - /0235
+---
+
+# Batch Delete
+
+Some APIs need to allow users to delete a set of resources in a single
+transaction. A batch delete method provides this functionality.
+
+## Guidance
+
+Batch delete methods are specified using the following pattern:
+
+```proto
+rpc BatchDeleteBooks(BatchDeleteBooksRequest) returns (google.protobuf.Empty) {
+  option (google.api.http) = {
+    post: "/v1/shelves/-/books:batchDelete"
```

Comment:
And again. (Just highlighting these to make sure we don't miss them after making the decision once.)